Provider crashes if objectstorage instance has no keys

If an objectstorage instance is created and the default key is removed by the user, a crash in the plugin can be triggered by creating a new `glesys_objectstorage_credential` resource referencing said instance.

Example crash:

```
glesys_objectstorage_instance.tvl-state: Refreshing state... [id=os-96e7b]
glesys_objectstorage_instance.tvl-backups: Refreshing state... [id=os-96d77]
╷
│ Error: Plugin did not respond
│ 
│   with glesys_objectstorage_instance.tvl-state,
│   on main.tf line 25, in resource "glesys_objectstorage_instance" "tvl-state":
│   25: resource "glesys_objectstorage_instance" "tvl-state" {
│ 
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ReadResource call. The plugin logs may
│ contain more details.
╵

Stack trace from the terraform-provider-glesys_v0.3.1 plugin:

panic: runtime error: index out of range [0] with length 0

goroutine 82 [running]:
github.com/glesys/terraform-provider-glesys/glesys.resourceGlesysObjectStorageInstanceRead(0xc00028f000, 0xc16120, 0xc000753dd0, 0x12052a0, 0xc000600000)
        github.com/glesys/terraform-provider-glesys/glesys/resource_glesys_objectstorage_instance.go:78 +0x454
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0xc00030e8c0, 0xda04d8, 0xc0001f9100, 0xc00028f000, 0xc16120, 0xc000753dd0, 0x0, 0x0, 0x0)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.8.0/helper/schema/resource.go:335 +0x1ee
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc00030e8c0, 0xda04d8, 0xc0001f9100, 0xc000209e10, 0xc16120, 0xc000753dd0, 0xc000116590, 0x0, 0x0, 0x0)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.8.0/helper/schema/resource.go:624 +0x1cb
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadResource(0xc0002b2060, 0xda04d8, 0xc0001f9100, 0xc0001f9140, 0xc0001f9100, 0x40d525, 0xc459a0)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.8.0/helper/schema/grpc_provider.go:576 +0x47d
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ReadResource(0xc0005942c0, 0xda0580, 0xc0001f9100, 0xc000115020, 0xc0005942c0, 0xc000283d40, 0xc000092ba0)
        github.com/hashicorp/terraform-plugin-go@v0.4.0/tfprotov5/tf5server/server.go:298 +0x105
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadResource_Handler(0xc74060, 0xc0005942c0, 0xda0580, 0xc000283d40, 0xc000114fc0, 0x0, 0xda0580, 0xc000283d40, 0xc000056380, 0xdb)
        github.com/hashicorp/terraform-plugin-go@v0.4.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:344 +0x214
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0002a6540, 0xda75f8, 0xc000102d80, 0xc000526000, 0xc00010e7e0, 0x11c8a90, 0x0, 0x0, 0x0)
        google.golang.org/grpc@v1.42.0/server.go:1282 +0x52b
google.golang.org/grpc.(*Server).handleStream(0xc0002a6540, 0xda75f8, 0xc000102d80, 0xc000526000, 0x0)
        google.golang.org/grpc@v1.42.0/server.go:1616 +0xd0c
google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc0000464e0, 0xc0002a6540, 0xda75f8, 0xc000102d80, 0xc000526000)
        google.golang.org/grpc@v1.42.0/server.go:921 +0xab
created by google.golang.org/grpc.(*Server).serveStreams.func1
        google.golang.org/grpc@v1.42.0/server.go:919 +0x1fd

Error: The terraform-provider-glesys_v0.3.1 plugin crashed!
```

--- a/glesys/resource_glesys_objectstorage_instance.go
+++ b/glesys/resource_glesys_objectstorage_instance.go
@@ -75,11 +75,14 @@ func resourceGlesysObjectStorageInstanceRead(d *schema.ResourceData, m interface
 	d.Set("datacenter", instance.DataCenter)
 	d.Set("description", instance.Description)
 	d.Set("created", instance.Created)
-	d.Set("accesskey", instance.Credentials[0].AccessKey)
 
-	_, secretkeyIsSet := d.GetOk("secretkey")
-	if !secretkeyIsSet {
-		d.Set("secretkey", instance.Credentials[0].SecretKey)
+	creds := instance.Credentials
+	if len(creds) > 0 {
+		d.Set("accesskey", creds[0].AccessKey)
+		_, secretkeyIsSet := d.GetOk("secretkey")
+		if !secretkeyIsSet {
+			d.Set("secretkey", creds[0].SecretKey)
+		}
 	}
 
 	return nil

